packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx

```diff
@@ -27,6 +28,8 @@ export interface WfoPageHeaderProps {
     handleLogoutClick: () => void;
 }
 
+const WEBSOCKET_FEATURE_TOGGLE = true;
```

Comment:
Think we want this stuck at false since it does not work currently, or how about to only show the badge when an actual env var is set?